tasks/sep-dev-0/006-holocene-sys-cfg/README.md

```diff
@@ -0,0 +1,31 @@
+# Holocene Hardfork - Contract Upgrades
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+Upgrades the `SystemConfig` contract for the Holocene hardfork.
+
+Contracts upgraded are included within the
+[op-contracts/v1.8.0-rc.1](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.8.0-rc.1) release.
+
+## Pre-deployments
+
+- `SystemConfig` - `0x29d06Ed7105c7552EFD9f29f3e0d250e5df412CD`
```

Comment:
We're actually skipping quite a few versions of `SystemConfig` with this upgrade.  It seems the devnet hasn't been updated in the past.  So I think we will need to add a reinitialize step to the upgrade for devnet so it sets things like the dispute game factory address which it doesn't currently have. The current version appears to be pre the MCP upgrade.

Reply:
Updated to set all of the MCP slots. The fee scalar slots and EIP-1559 parameter slots can be left alone, the old values will be used.